modules/openideal_idea/js/openideal_idea_useful.js

```diff
@@ -12,7 +12,12 @@
           $this.find('[type=submit]').hide();
           var $select = $(this);
           var isPreview = $select.data('is-edit');
-          $select.after('<div class="like-rating"><a href="#"><i class="fa fa-thumbs-up"></i></a></div>').hide();
+          // Depending on vote entity change the body.
+          var likeBody = settings.openidealUser.comment
+          ? '<i class="fa fa-thumbs-up"></i>'
+          : '<span class="mr-2"><i class="fa fa-thumbs-up"></i></span> RATE IDEA';
```

Comment:
"Rate idea" should be wrapped into t() function. Uppercase should be controlled on CSS level.